### PR TITLE
Fix: config validation before starting the service avoids panic

### PR DESCRIPTION
**Pull request by guacamole**
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -107,6 +107,42 @@ func NewStoreConfig() (*Store, error) {
...
 
+func (oc *OpenRegistryConfig) Validate() error {
+	if oc == nil {
+		return fmt.Errorf("invalid config, cannot be nil")
+	}
+	v := validator.New()
+
+	english := en.New()
+	uni := ut.New(english, english)
+	trans, _ := uni.GetTranslator("en")
```
Comment (jay-dee7) [inline config/config.go:118]: @guacamole shouldn't we be checking these errors?

File: config/config.go [hunk 2]
```diff
@@ -107,6 +107,42 @@ func NewStoreConfig() (*Store, error) {
...
+		validatorErrs, ok := err.(validator.ValidationErrors)
+		if !ok {
+			return err
+		}
+		for _, e := range validatorErrs {
+			translatedErr = multierror.Append(translatedErr, fmt.Errorf(e.Translate(trans)))
+		}
+		return translatedErr
+	}
+	return nil
```
Comment (jay-dee7) [inline config/config.go:143]: @guacamole here we should return as error as is

Comment (jay-dee7) [reply config/config.go:143]: no this is correct

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
@guacamole Just few changes and we're good

File: config/config.go (hunk 1 above)
Comment (guacamole) [reply config/config.go]: you're right

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 🥇 🥇

**Pull request merged**